README.md

```diff
@@ -39,31 +50,39 @@ out_data = make_sets(
 ```
 You can generate the prompts by:
 ```python
-from setlexsem.generate.generate_prompts import generate_prompts
+from setlexsem.generate.generate_prompts import create_prompts
 
-generate_prompts(
+data_config = {
+    "set_types": ["words"],
+    "n": [1000],
+    "m": [2, 16],
+    "item_len": [2],
+    "decile_group": [5],
+    "swap_status": None,
+    "overlap_fraction": [0.5],
+}
+
+prompt_config = {
+    "op_list": ["union", "intersection"],
+    "k_shot": [0, 1, 3],
+    "prompt_type": ["formal_language"],
+    "prompt_approach": ["baseline", "chain_of_thought"],
+    "is_fix_shot": [True]
+}
+
+prompt_and_ground_truth = create_prompts(
     # data config
-    set_types=["numbers"],
-    n=[10],
-    m=[1,2],
-    item_len=[1,3],
-    decile_group=None,
-    swap_status=None,
-    overlap_fraction=[None],
-    number_of_data_points= 5,
+    data_config=data_config,
+    number_of_data_points=5,
     random_seed_value=292,
     # prompt config
-    op_list=["union", "intersection"],
-    k_shot=[0, 1, 3],
-    prompt_type=["formal_language"],
-    prompt_approach=["baseline", "chain_of_thought"],
-    is_fix_shot=[True],
+    prompt_config=prompt_config,
     add_roles=False)
 ```
 
 
 
-# Install
+# Development
 
```

Comment:
So now we have a section called `Development` that starts by talking about installing?